### PR TITLE
nolo tlc ciel 9_3_1 to 9_3_2

### PR DESCRIPTION
**Pull request by nolo16**
torture is cool... i guess?

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_3_1.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [sha:5c739aa6b10dc78abb740cc9ad4c4fd7d6ee3733]{
 -- Page 10009, Offset 22962.
 -- 「なに？　やっぱり、シエルさんと学生ごっこやってる志貴クンには荷が重い？」
-"What? Is that too much for someone like you, pretending to be a student like Ciel-san?"
+"What? Is it too much for someone who's playing make-believe student like Ciel-san, after all?"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_3_1.txt:4]: "like Ciel-san" sounds like Ciel is the one she's talking about, when in fact she's just comparing the two. My take is a little bit modified from the source but makes more sense in EN
> What? Too much to bear for you? Still playing make-believe student like Ciel-san?

File: script/Ciel/Day 9/09_01_CIEL09_3_1.txt [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 }
 [sha:59aaad6a62f7eb4a11e0e88f8217458484206ba4]{
 -- Page 10009, Offset 22963.
 -- 挑発とも批難とも取れる女の声。
-I can't say whether the woman is trying to provoke me or to criticize me.
+I can't tell if she's trying to provoke me or criticize me.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_3_1.txt:9]: I can't tell from her voice if she's trying to provoke me or criticize me.

File: script/Ciel/Day 9/09_01_CIEL09_3_2.txt [hunk 3]
```diff
@@ -11,27 +11,27 @@ I swat her hand away.
 [sha:4dbe5824d8bedb5c47e47876286a89da2cab537d]{
 -- Page 10010, Offset 23023.
 -- 振り払った手の勢いが思いの<外|ほか>強かったのは、女への反感によるものだろう。
-I swing my hand in an unexpectedly forceful way, but that must be the reflection of my distaste for the woman.
+I ended up swinging my hand more forcefully than I expected. I guess it's a reflection of my distaste for the woman.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_3_2.txt:14]: "for the woman" is very awkward. "distaste for her"

File: script/Ciel/Day 9/09_01_CIEL09_3_2.txt [hunk 4]
```diff
@@ -216,22 +216,22 @@ Just listening to his supplications is enough to deeply pain a man's heart.
...
 -- Page 360, Offset 23064.
 -- あるいは、ソレは本気で更生しようと誓ったかもしれない。
-He may be vowing in all seriousness to repent himself.
+He may be earnestly vowing to reform himself.
 }
 [sha:f1a1cdefef841a467082fbf7da7519699674a096]{
 -- Page 360, Offset 23065.
 -- この悪夢、目の前の女に生きたまま■■される悪夢から抜け出せるなら罪を償うと。
-Vowing to atone for his sins if he can break free from this nightmare, the nightmare of being ■■ed while alive by the woman in front of him.
+Vowing to atone for his sins if he can break free from this nightmare, the nightmare of being ■■ed alive by the woman in front of him.
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_3_2.txt:224]: I want to be ■■ed alive by Noel too :roa:

File: script/Ciel/Day 9/09_01_CIEL09_3_2.txt [hunk 5]
```diff
@@ -341,7 +341,7 @@ The upper half of the now freed vampire begins to twitchingly crawl.
 [sha:59153614d2487d309fe0325f64792064b4107b6e]{
 -- Page 368, Offset 23089.
 -- 「ほら、見てよこのアリサマ！　ゴキブリみたい！　こんなになってもコイツらは死なないの！　気持ち悪い、気持ち悪い、気持ち悪い、気持ち悪い……！　こんなの生きてる価値もない！」
-"There, look at him! He's just like an cockroach! Even reduced to this state, they still won't die! It's disgusting, disgusting, disgusting, disgusting...! They don't deserve to live!"
+"There, look at it! It's just like a cockroach! Even when reduced to this state, they still won't die! It's disgusting, disgusting, disgusting, disgusting...! Vampires don't deserve to live!"
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_3_2.txt:344]: > 見てよこのアリサマ
look how he ended up!

**Review by Avuxo — APPROVED** [review]
tsukihimates does not endorse torture.




or do we?

File: script/Ciel/Day 9/09_01_CIEL09_3_2.txt (hunk 4 above)
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_3_2.txt:224]: image

**Pull request merged**